.gitignore

```diff
@@ -7,5 +7,4 @@ spec/dummy/db/*.sqlite3
 spec/dummy/config/deprecations.ignore
 doc/
 .yardoc/
-Gemfile.lock
 /gemfiles/*.gemfile.lock
```

Comment:
let's remove this as well, since CI runs against the various appraisal gemfiles (and commit the gemfile.lock for each appraisal)

Reply:
Ok, did this. I unpinned `appraisal` (see: https://github.com/thoughtbot/appraisal/issues/199#issuecomment-1374700828; tldr `be appraisal install` didn't work with the version we had)